brainscore_language/plugins/pereira2018/benchmark.py

```diff
@@ -0,0 +1,100 @@
+from brainscore_core.metrics import Score
+from brainscore_language import load_dataset, load_metric, benchmarks
+from brainscore_language.artificial_subject import ArtificialSubject
+from brainscore_language.utils.ceiling import ceiling_normalize
+from brainscore_language.utils.s3 import load_from_s3
+from .data import BIBTEX
+
+logger = logging.getLogger(__name__)
+
+
+def Pereira2018_243sentences():
+    return _Pereira2018ExperimentLinear(experiment='243sentences', ceiling_s3_kwargs=dict(
+        version_id='CHl_9aFHIWVnPW_njePfy28yzggKuUPw',
+        sha1='5e23de899883828f9c886aec304bc5aa0f58f66c',
+        raw_kwargs=dict(
+            version_id='uZye03ENmn.vKB5mARUGhcIY_DjShtPD',
+            sha1='525a6ac8c14ad826c63fdd71faeefb8ba542d5ac',
+            raw_kwargs=dict(
+                version_id='XVTo58Po5YrNjTuDIWrmfHI0nbN2MVZa',
+                sha1='34ba453dc7e8a19aed18cc9bca160e97b4a80be5'
+            )
+        )
+    ))
+
+
+def Pereira2018_384sentences():
+    return _Pereira2018ExperimentLinear(experiment='384sentences', ceiling_s3_kwargs=dict(
+        version_id='sjlnXr5wXUoGv6exoWu06C4kYI0KpZLk',
+        sha1='fc895adc52fd79cea3040961d65d8f736a9d3e29',
+        raw_kwargs=dict(
+            version_id='Hi74r9UKfpK0h0Bjf5DL.JgflGoaknrA',
+            sha1='ce2044a7713426870a44131a99bfc63d8843dae0',
+            raw_kwargs=dict(
+                version_id='m4dq_ouKWZkYtdyNPMSP0p6rqb7wcYpi',
+                sha1='fe9fb24b34fd5602e18e34006ac5ccc7d4c825b8'
+            )
+        )
+    ))
+
+
+class _Pereira2018ExperimentLinear(BenchmarkBase):
+    """
+    Evaluate model ability to predict neural activity in the human language system in response to natural sentences,
+    recorded by Pereira et al. 2018.
+    Alignment of neural activity between model and human subjects is evaluated via cross-validated linear predictivity.
+
+    This benchmark builds off the behavioral benchmark introduced
+    in Schrimpf et al. 2021 (https://www.pnas.org/doi/10.1073/pnas.2105646118), but:
+
+    * computes neural alignment to each of the two experiments ({243,384}sentences) separately, as well as ceilings
+    * requires the model to have committed to neural readouts (e.g. layer 41 corresponds to the language system),
+        rather than testing every layer separately
+
+    Each of these benchmarks evaluates one of the two experiments, the overall Pereira2018-linear score is the mean of
+    the two ceiling-normalized scores.
+    """
+
+    def __init__(self, experiment: str, ceiling_s3_kwargs: dict):
+        self.data = self._load_data(experiment)
+        self.metric = load_metric('linear_pearsonr')
+        identifier = f'Pereira2018.{experiment}-linear'
+        ceiling = self._load_ceiling(identifier=identifier, **ceiling_s3_kwargs)
+        super(_Pereira2018ExperimentLinear, self).__init__(
+            identifier=identifier,
+            version=1,
+            parent='Pereira2018-linear',
```

Comment:
notation-wise, should we use "linear_pearsonr" or "linreg_pearsonr"? Aalok and I started using the latter, but I don't feel super strongly. "linear" just comes across as an umbrella term

Reply:
+1 for `linreg`

Reply:
My thinking was that we probably won't use `Spearman-r` for correlation of linear regression, so omitted the correlation type, but I'm open to the naming. As long as identifiers are unique and somewhat descriptive I'm happy

Reply:
created issue #41